src/toil/provisioners/gceProvisioner.py

```diff
@@ -852,12 +876,9 @@ def _waitForAppliance(cls, ip_address, role, keyName='core'):
                     logger.info('...Still waiting for appliance, trying again in %s sec...' % sleepTime)
                     time.sleep(sleepTime)
             except:
-                if retries == 5:
-                    raise # givup
```

Comment:
Why did you get rid of this?

Reply:
Time based failure seems to work better. That is covered by the loop on line 839. This mechanism seemed redundant or not very useful.